Azure/sonic-buildimage
# [Centec][arm64] support new board E530-48s4x and E530-24x2q

1. support new board E530-48s4x E530-24x2q
    2. optimize platform driver for Centec TsingMa board



#### Why I did it
1. Support new board E530-48s4x and E530-24x2q
2. Platform functions (fan, psu, sfp, etc)  improvement
3. Upgrade sai to v1.7

#### How I did it
Modify related code in platform and device directory.

#### How to verify it
Build centec arm64 sonic image,  verify platform functions (fan, psu, sfp, etc) on E530 series board.

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog



#### A picture of a cute animal (not mandatory but encouraged)
